package.json

```diff
@@ -23,9 +25,18 @@
     "@babel/core": "^7.13.10",
     "@babel/node": "^7.13.12",
     "@babel/preset-env": "^7.13.12",
-    "nodemon": "^2.0.7"
+    "@types/express": "^4.17.11",
+    "express": "^4.17.1",
+    "nodemon": "^2.0.7",
+    "tslint": "^6.1.3",
+    "typescript": "^4.2.3"
```

Comment:
Also, you have to add
`"@types/node": "^14.14.35",
    "@types/uuid": "^8.3.0",`